rdt-open-reader/src/main/java/io/ona/rdt/application/RDTApplication.java

```diff
@@ -74,6 +76,8 @@ public void onCreate() {
         if (StringUtils.isEmpty(sharedPreferences.getPreference(AllConstants.LANGUAGE_PREFERENCE_KEY))) {
             sharedPreferences.savePreference(AllConstants.LANGUAGE_PREFERENCE_KEY, BuildConfig.LOCALE);
         }
+
+        syncUtils = new SyncUtils(RDTApplication.getInstance());
```

Comment:
Why do we need to keep this here? Can't we have this in the Utils class?

Reply:
basically, i need to mock `SyncUtils` class that's why i have moved here

Reply:
why can't it be mocked elsewhere?

Reply:
moved on User `UserAuthorizationVerificationTask` class